Codecademy/gamut
# fix(Markdown): nested ordered and unordered list fix

### Overview


Fixes a bug with Markdown where ordered list items  that were nested in an unordered list were still getting a bullet point


### PR Checklist

- [x] Related to JIRA ticket: web-1983

#### Testing Instructions


* Navigate to the Markdown component
* Observe no visual bugs on the nested ordered/unordered lists
 
#### PR Links and Envs

**_Note_**: I am using these same PR envs as the [Markdown Checkbox PR](https://github.com/Codecademy/gamut/pull/2460) -- the alpha packages are this PR's (which branches off of Markdown Checkbox)

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/31680)  | [Monolith Env](https://pr-31680-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/2279)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2279)   |
